operations/finance.md

```diff
@@ -295,7 +295,8 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 | `Client` | string | required | Name and version of the client application. |
 | `ConsumedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the accounting item was consumed. Required if no other filter is provided. |
 | `ClosedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the accounting item was closed. Required if no other filter is provided. |
-| `ItemIds` | array of string | optional, max 1000 items | Unique identifiers of the Accounting items. Required if `ConsumedUtc` or `ClosedUtc` are not specified. |
+| `ItemIds` | array of string | optional, max 1000 items | Unique identifiers of the Accounting items. Required if `RebatedItemIds` or `ConsumedUtc` or `ClosedUtc` are not specified. |
+| `RebatedItemIds` | array of string | optional, max 1000 items | Unique identifiers of the Accounting items we are finding rebates for. Required if `ItemIds` or `ConsumedUtc` or `ClosedUtc` are not specified. |
```

Comment:
@vitasamek 
1. Should we just say `Required if no other filter is provided.` like for ClosedUtc for example?
2. I feel like if you don't know what rebates are, this isn't really helpful, but this is not the place to be explaining it. Any ideas? Or rebates are edge case for API usage anyway, so no point in explaining?

Reply:
1) Makes sense, maybe rewrite it everywhere to `Required if no other filtering is requested`? This could be more user oriented than us oriented.
2) Discussed on call, we should add rebate use case which will explain the whole flow how to obtain the info.